### DISCUSSION TITLE
Merge Ubi Connect from NSL with that installed by Steam?

### DISCUSSION THREAD

**Discussion by Siegfriedmk** [moraroy/NonSteamLaunchers-On-Steam-Deck#109] [Q&A]
Hello.
I installed Ubisoft Connect via Non Steam Launcher, to install some games I only have on Connect, and it works fine. Added to Gaming mode and all.

Now, I installed Immortal Fenyx Rising on Steam, which also launch Connect, but of course it doesn't detect the other installation of Connect and the games I already downloaded on the "other" Connect. It's a little messy situation. Is it possible to merge them both?

**Comment by moraroy** [comment]
@Siegfriedmk if you have a shortcut in your steam library for Fenyx Rising, go inside its launch options in the steam shortcut and copy the same launch options from the NSL Ubisoft into the Fenix rising one. I believe your steam game will then use the NSL ubisoft. What this means is you may have to sign in, you have no game saves, you may be starting "new" on that game. But this obviouy also depends on a game to game basis.
